graphql_client/__init__.py

```diff
@@ -87,9 +87,11 @@ def subs(_cc):
         return _id
 
     def stop_subscribe(self, _id):
-        self._subscription_running = False
-        self._st_id.join()
-        self._stop(_id)
+        if self._subscription_running:
```

Comment:
@BharathaAravind can you add some comments explaining what is the exact issue it solves?